.storybook/webpack.config.js

```diff
@@ -11,6 +17,20 @@ module.exports = (storybookBaseConfig, env, config) => {
                     appendTsSuffixTo: [/\.vue$/],
                     transpileOnly: true
                 }
+            }, {
+                test: svgRule.test,
+                oneOf: [{
+                    // Add new icon SVG rule
+                    test: /icons/,
+                    loader: 'svg-sprite-loader',
+                    options: {
+                        symbolId: filePath => `nq-${path.basename(filePath, '.svg')}`
+                    }
+                }, {
+                    // Re-add default SVG rule
+                    loader: svgRule.loader,
+                    options: svgRule.options
+                }],
```

Comment:
Both storybook and vue-cli have their own default webpack config, and yes, both include a default SVG handler (the `file-loader` loader). Since the icon SVGs need to be handled differently than, say the identicon SVG, I had to change that default rule. Only adding a new rule would have executed both for the icons, which would still export them and actually broke the sprite loader.

At first (first commit) I dublicated the default SVG loader code in the webpack config files, and in the second commit I found that I could just re-use the default one, so it's more future proof if the default ever changes.